### DISCUSSION TITLE
How to use `doppler` in Prisma schema

### DISCUSSION THREAD

**Discussion by l2D** [prisma/prisma#9913] [Q&A] [answered]
I would like to know how to use [doppler](https://docs.doppler.com/docs/enclave-installation) in `schema.prisma`

**Comment by austincrim** [comment, marked as answer]
Hey @l2D,

To run locally, follow [Doppler's instructions](https://docs.doppler.com/docs/enclave-installation#local-development) to get your environment variables set when running your Node process. Then, ensure your environment variable name in Doppler matches what is in your `env()` in your Prisma Schema's `datsource` block.

When deploying, follow Doppler's various integration guides [here](https://docs.doppler.com/docs/integrations).

**Reply by l2D** [reply]
Thank you for your help.

I don't know why this case is not working but when I changed variables names to `DATABASE_URL` and `SHADOW_DATABASE_URL` it's work fine. 

<details>
<summary>Image</summary>

Not working case:
![schema prisma — we-nodejs_base_2021-10-23_05-03-20_rjpgXkAZ](https://user-images.githubusercontent.com/20911264/138527916-cea468d0-a5a9-4c62-a185-0caec59207c4.png)

Working case:
![schema prisma — we-nodejs_base_2021-10-23_05-11-03_XwLdk4Wk](https://user-images.githubusercontent.com/20911264/138528444-6c6986cb-ed09-47be-b764-495dd86f2228.png)


</details>

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
